src/server/controllers/Grbl/GrblController.js

```diff
@@ -373,7 +373,8 @@ class GrblController {
                     });
                 }
 
-                return line;
+                // remove all comments before forwarding on
+                return line.replace(/([^\)]*)/gm, '').replace(commentMatcher, '');
```

Comment:
Change this to return line.replace(/\([^\)]*\)/gm, '').trim();

Reply:
https://github.com/Sienci-Labs/gsender/pull/448/commits/c609723ecffe93c6ce104906b192a6f8be2ffdea